Hunting Queries/SecurityEvent/ADAccountLockouts.yaml

```diff
@@ -0,0 +1,21 @@
+id: e7642e6e-cf27-46ec-a4b9-e4475228fead
+name: AD Account Lockout
+description: |
+  'Detects Active Directory account lockouts'
+requiredDataConnectors:
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvent
+tactics:
+  - InitialAccess
+  - CredentialAccess
+relevantTechniques:
+  - T1531
+query: |
+let timeframe = 7d;
```

Comment:
2 spaces before each line in the query, otherwise the YAML template format is wrong and the build checks break.